data/json/items/comestibles/egg.json

```diff
@@ -1169,5 +1151,105 @@
     "material": [ "egg", "veggy" ],
     "fun": 5,
     "flags": [ "EATEN_HOT", "FREEZERBURN" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_proxy_frog",
+    "copy-from": "egg_chicken",
```

Comment:
This is both good and bad. You're copying stuff from an egg which makes sense - but I am 99% sure caloric contents would be different and barter value should definitely be lower. If anything I'd advise copying from roe and adjusting price to be lower. Also, while I'm here, are you sure those are safe to consume? I vaguely recall hearing toad eggs specifically were poisonous?